runtime/service_integ_test.go

```diff
@@ -728,3 +729,120 @@ func TestCreateTooManyContainers_Isolated(t *testing.T) {
 	assert.Equal("no remaining stub drives to be used: unavailable: unknown", err.Error())
 	require.Error(t, err)
 }
+
+func TestDisableJailing_Isolated(t *testing.T) {
```

Comment:
nit: can we move this to `jailer_integ_test.go` or similar? This file is starting to get overwhelming. I was similarly planning on putting the new cni tests in `cni_integ_test.go`